DotNetHowToEncryptionUsingCMK/DotNetHowToEncryptionUsingCMK/Program.cs

```diff
@@ -190,129 +188,30 @@ private static void UploadDocuments(ISearchIndexClient indexClient)
-                        new Hotel()
-                        {
-                            HotelId = "2",
-                            BaseRate = 79.99,
-                            Description = "Cheapest hotel in town",
-                            DescriptionFr = "Hôtel le moins cher en ville",
-                            HotelName = "Roach Motel",
-                            Category = "Budget",
-                            Tags = new[] { "motel", "budget" },
-                            ParkingIncluded = true,
-                            SmokingAllowed = true,
-                            LastRenovationDate = new DateTimeOffset(1982, 4, 28, 0, 0, 0, TimeSpan.Zero),
-                            Rating = 1,
-                            Location = GeographyPoint.Create(49.678581, -122.131577)
-                        }),
-                    IndexAction.MergeOrUpload(
-                        new Hotel()
-                        {
-                            HotelId = "3",
-                            BaseRate = 129.99,
-                            Description = "Close to town hall and the river"
-                        }),
-                    IndexAction.Delete(new Hotel() { HotelId = "6" })
-                };
-
-            var batch = IndexBatch.New(actions);
-
-            try
-            {
-                indexClient.Documents.Index(batch);
-            }
-            catch (IndexBatchException e)
-            {
-                // Sometimes when your Search service is under load, indexing will fail for some of the documents in
-                // the batch. Depending on your application, you can take compensating actions like delaying and
-                // retrying. For this simple demo, we just log the failed document keys and continue.
-                Console.WriteLine(
-                    "Failed to index some of the documents: {0}",
-                    String.Join(", ", e.IndexingResults.Where(r => !r.Succeeded).Select(r => r.Key)));
-            }
-
-            Console.WriteLine("Waiting for documents to be indexed...\n");
-            Thread.Sleep(2000);
-        }
-#endif
-
         private static void RunQueries(ISearchIndexClient indexClient)
         {
             SearchParameters parameters;
             DocumentSearchResult<Hotel> results;
 
-            Console.WriteLine("Search the entire index for the term 'budget' and return only the hotelName field:\n");
-
-            parameters =
-                new SearchParameters()
-                {
-                    Select = new[] { "hotelName" }
-                };
-
-            results = indexClient.Documents.Search<Hotel>("budget", parameters);
-
-            WriteDocuments(results);
-
-            Console.Write("Apply a filter to the index to find hotels cheaper than $150 per night, ");
-            Console.WriteLine("and return the hotelId and description:\n");
+            Console.WriteLine("Search with terms nonexistent in the index:\n");
```

Comment:
Maybe this should move down and replace the message about searching for "five star" (since that message just parrots the code anyway).

Reply:
why? it explains that the next queries ('five start', 'internet') are intentionally not in the indexed documents, but rather synonyms of other indexed terms

Reply:
I didn't realize it was for the whole group; nevermind then